book/website/reproducible-research/renv/renv-virtualmachine.md

```diff
@@ -289,6 +289,6 @@ You can also destroy the virtual machine to clean up all storage associated with
 
 ```{code-block} console
 [vagrant@theturingway ~]$ exit
```

Comment:
Was this one intentionally kept?

Reply:
Hmmm - I remember seeing this one and not being sure what to do with it...

Do you have an opinion @da5nsy ? Is there a way that this looks the most clear?